### PR TITLE
Contribution certifications

### PR DESCRIPTION
**Pull request by advayDev1**
Note: This is not legal advice.  Contributors, users, etc. must ensure their own level of comfort with
contributions certified as described here, and should seek their own legal counsel as needed.

See the License for disclaimers and limitations of liability, including but not limited to
Section 7:

```
7. Disclaimer of Warranty. Unless required by applicable law or
      agreed to in writing, Licensor provides the Work (and each
      Contributor provides its Contributions) on an "AS IS" BASIS,
      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
      implied, including, without limitation, any warranties or conditions
      of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A
      PARTICULAR PURPOSE. You are solely responsible for determining the
      appropriateness of using or redistributing the Work and assume any
      risks associated with Your exercise of permissions under this License.
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,6 +15,40 @@ the [Apache 2.0 License](LICENSE).
 [preparing-your-pull-request-for-submission](#preparing-your-pull-request-for-submission).
 
 
+### Conditions for accepting pull requests
+Please confirm that you can certify the following,
```
Comment (brunobowden) [inline CONTRIBUTING.md]: Any reason for the strange line wrap?

**Comment by brunobowden** [comment]
Can we make it apply to all future contributions, just to make it easier? It would then act closer to a normal CLA.

**Comment by brunobowden** [comment]
LGTM either way

**Comment by advayDev1** [comment]
I don't know how we'd make it apply to future contributions; as different git repos/branches/forks could have different sequences of commits.  I think we should keep it simpler, and if/when we figure out Apache-style CLAs just use those.

File: CONTRIBUTING.md (hunk 1 above)
Comment (advayDev1) [inline CONTRIBUTING.md]: i'm just going to leave it alone to avoid repushing :)

**Pull request merged**